pkg/util/parsers/parsers.go

```diff
@@ -56,3 +56,45 @@ func ParseImageName(image string) (string, string, string, error) {
 	}
 	return repoToPull, tag, digest, nil
 }
+
+// ApplyDefaultImageTag parses a docker image string, if it doesn't contain any tag or digest,
+// a default tag will be applied.
+func ApplyDefaultImageTag(image string) (string, error) {
+	named, err := dockerref.ParseNormalizedNamed(image)
```

Comment:
Can `dockerref` be used for other container runtime? Does OCI define that?

Reply:
It's how kubelet assumes , we definitely have to change this in the future. OCI barely play a role in kubelet image manager right now, it's sad.